lms/djangoapps/verify_student/migrations/0016_verificationattempt_status_changed.py

```diff
@@ -0,0 +1,20 @@
+# Generated by Django 4.2.15 on 2024-09-19 16:17
```

Comment:
If possible, could you squash these migrations, please? It keeps things tidy.

Reply:
Done!